pkg/osdCloud/aws.go

```diff
@@ -237,48 +238,50 @@ func CreateAWSV2Config(ctx context.Context, clusterID string) (awsv2.Config, err
 
 // GetAWSClientInputFromBackplane sets up AWS credentials via backplane-api given a cluster id
 func GetAWSClientInputFromBackplane(clusterID string) (*aws.AwsClientInput, error) {
-	token, err := utils.GetOCMApiServerToken()
-
-	getUrl, err := utils.GetBackplaneURL(clusterID)
-	if err != nil {
-		return nil, fmt.Errorf("Unable to retrieve backplane URL for cluster %s: %s", clusterID, err)
+	ocmToBackplane := map[string]string{
+		"https://api.openshift.com":             "https://api.backplane.openshift.com",
+		"https://api.stage.openshift.com":       "https://api.stage.backplane.openshift.com",
+		"https://api.integration.openshift.com": "https://api.integration.backplane.openshift.com",
```

Comment:
Could we get those URL from the backplane config? we are intent to not expose those URLs in public.

Reply:
Definitely, I would like to - I think what we are missing currently is helper functions to read the backplane config, but will look into writing some in backplane-cli

Reply:
I tested the new functions I added in https://github.com/mjlshen/mirrosa/pull/138 (I'll port that over here as I think it satisfies this request!)